supertokens_python/recipe/session/recipe_implementation.py

```diff
@@ -171,8 +171,8 @@ async def validate_claims_in_jwt_payload(
 
     async def get_session(
         self,
-        access_token: str,
-        anti_csrf_token: Optional[str],
+        access_token: Optional[str] = None,
```

Comment:
I don't think this matters too much, but this is not an optional parameter. It can be undefined but it has to be set to undefined explicitly.

Reply:
done